### PR TITLE
Moodle-LMS.md

### PR DESCRIPTION
**Pull request by Poojith-Obeysekara**

### CODE REVIEW

File: Investigations/Moodle-LMS.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+Moodle Learning Management System
+
+=> Introduction
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]:
```suggestion
## Introduction
```

File: Investigations/Moodle-LMS.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+* Moodle also known as (Modular Object-Oriented Dynamic Learning Environment) is a free and open source online Learning Management System that allows instructors to establish their own private website with dynamic courses that allow students to learn at their own pace, anywhere at any time.
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]: Just put it as normal paragraph

File: Investigations/Moodle-LMS.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+ 
+* Since Moodle is one of the top open source Learning Management Systems available in the world most of the higher education institutions use it as their main eLearning platform. Moodle is written in PHP and distributed under the GNU General Public Licence.
+ 
+* Moodle is developed on pedagogical principles. So that it can be used not only at universities and higher education institutions but also for blended learning, distance learning, flipped classrooms, etc.
+
+
+
+
+=> History
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]:
```suggestion
## History
```

File: Investigations/Moodle-LMS.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
+Moodle Learning Management System
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]:
```suggestion
# Moodle Learning Management System
```

**Review by dhanushkac — CHANGES_REQUESTED** [review]

File: Investigations/Moodle-LMS.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+### 5) Install Moodle
+
+Go to the URL for your moodle site in a browser (installation will complete automatically) or run the command line version at (requires cli version of PHP):
+/usr/bin/php /path/to/moodle/admin/cli/install.php
+ 
+The CLI creates the config.php for you and will not run if you created one in the previous step.
+After completing the install make sure your file permissions are ok for the Moodle program files (not writeable by web server) and the Moodle data files (writeable by web server).
+Set up cron
+You will need a cron job to run periodically. It is recommended that the cron is run every minute, as required for asynchronous activity deletion when using the recycle bin. A typical Unix cron entry will be as follows:
+* * * * *    /usr/bin/php /path/to/moodle/admin/cli/cron.php >/dev/null
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]:
```suggestion
`/usr/bin/php /path/to/moodle/admin/cli/cron.php >/dev/null`
```

File: Investigations/Moodle-LMS.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+Your site will not work properly unless cron is running regularly. It is very important you do not skip this step.
+ 
+
+
+**Congratulations!**
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]: I think this part is not needed

File: Investigations/Moodle-LMS.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+You are now ready to use your Moodle site. 
+If you run into problems, check the Installation FAQ and visit the Installation help forum.
+
+---
+
+(University : ICBT Kandy Campus)
```
Comment (dhanushkac) [inline Investigations/Moodle-LMS.md]: We can omit this as well

**Review by dhanushkac — CHANGES_REQUESTED** [review]

File: Investigations/Moodle-LMS.md (hunk 6 above)
Comment (Poojith-Obeysekara) [reply Investigations/Moodle-LMS.md]: Ok aiya

File: Investigations/Moodle-LMS.md (hunk 7 above)
Comment (Poojith-Obeysekara) [reply Investigations/Moodle-LMS.md]: Sure aiya, I'll do so

**Review by dhanushkac — APPROVED** [review]
👍 LGTM

**Pull request merged**